x-pack/plugins/enterprise_search/public/applications/shared/layout/nav.tsx

```diff
@@ -80,34 +78,26 @@ export const useEnterpriseSearchNav = () => {
       }),
     },
     {
-      id: 'enterpriseSearchAnalytics',
+      id: 'enginesSearch',
```

Comment:
Nit: If this is new should it be `searchApplicationsSearch`?

Reply:
We have other tickets for renaming so I was trying to keep changes limited here.

Reply:
renamed this to just `search` now, but there will be another issue coming soon to do some refinements on the nav for 8.8